saiku-webapp/src/main/resources/ccrs-mondrian-cp1.xml

```diff
@@ -247,6 +247,44 @@
 			</Row>
 		</Rows>
 	</InlineTable>
+	
+        <Query alias="CP1UncgMember">
+            <ExpressionView>
+                <SQL dialect="generic">
+                    <![CDATA[
+                        SELECT
+                            M.*, CP1.DESIGNATION, CP1.ENTITY_YEAR, CP1.COUNTRY_INCOME_STATUS_ID, CP1.SECONDARY_COUNTRY_INCOME_STATUS_ID,
+                            CASE WHEN cp1.OFFICIAL_REQUEST_DAO_COUNTRY = TRUE THEN 1 ELSE 0 END as OFFICIAL_REQUEST_DAO_COUNTRY
+                        FROM
+                            CP1UNCGMEMBER M, CP1GENERAL_PROFILE CP1, BASE_COUNTRY bc, V_CP_VALIDATED_DESIGNATIONS v
+                        WHERE
+                            M.PARENT_ID = CP1.ID AND
+                            CP1.DESIGNATION != 'AQ' AND CP1.DESIGNATION != 'DOCO' AND CP1.DESIGNATION = bc.ISO2 AND
+                            CP1.DESIGNATION = v.DESIGNATION AND v.ENTITY_YEAR = CP1.ENTITY_YEAR
+                    ]]>
+                </SQL>
+            </ExpressionView>
+        </Query>
+        
+        <InlineTable alias="DAOCP1UncgMember">
```

Comment:
I think we will have to consolidate all tables similar to this and name it something like boolean_to_text and reuse. I think this will not change anything in output but will decrease repetition in xml.

Reply:
Agree, but I just followed consistent approach based on preexisting definitions. Check the original CP1 definition and you'll see lots of exactly the same inline boolean tables, e.g. DAOCP1GeneralCube, DAO_VISION_YES_NO, HAS_ANNUAL_REVIEW, REPORT_SUBMITTED_TO_GOV, BUDGETED_WORKPLAN, etc. I've seen the same in other CPx schema.

As I mentioned to you before, I would like to optimize the definitions, and not only for this table. This should be done in a separate task, here is the ticket I created for it: [CCR-1203](https://jira.developmentgateway.org/browse/CCR-1203)